SweetTweetApp/src/main/java/users/UserCollection.java

```diff
@@ -0,0 +1,74 @@
+package users;
+
+import org.apache.commons.codec.digest.DigestUtils;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class UserCollection {
+
+    private static HashMap<String, User> users = new HashMap<>();
+    public static int availableId = 2;
+
+    public static String getHash(String password) {
+        return new DigestUtils("SHA-256").digestAsHex(password);
+    }
+
+    public static boolean isHashEqual(String password, String passwordHash) {
+        String passwordForCheck = new DigestUtils("SHA-256").digestAsHex(password);
```

Comment:
I'll just add some words for the future. Nowadays there are lots of tables with the most popular strings hashed with different algorithms. So, if an attacker gets database, he can get at least access to users with weak passwords. To make harder to retrieve password, usually together with hashing salt is used (just some random addition), that increases complexity of decrypting passwords. If you are interested, you can read more about it, if not - just ignore that message 🙂